eurofaang_ri_be/eurofaang_ri_be/settings.py

```diff
@@ -44,6 +48,7 @@
     'rest_framework.authtoken',
     'corsheaders',
     'users',
+    'phonenumber_field'
```

Comment:
do we really need additional dependency here just for phone number field, I think simple RE will work - number must start with +sign and then have numbers